solace-spring-cloud-starters/solace-spring-cloud-stream-starter/README.adoc

```diff
@@ -872,6 +874,43 @@ See <<Native Payload Types>> for more info regarding this binder's natively supp
 
 To create a batch of messages, the binder will consume messages from the PubSub+ broker until either a maximum batch size or timeout has been achieved. After which, the binder will compose the batch message and send it to the consumer handler for processing. Both these batching parameters can be configured using the `batchMaxSize` and `batchTimeout` consumer config options.
 
+=== Batch Producers
```

Comment:
Should CorrelationData be mentioned somewhere? That is, one CorrelationData per batch.

Reply:
done